core/app/models/spree/product.rb

```diff
@@ -184,13 +184,13 @@ def property(property_name)
       product_properties.find_by(property: prop).try(:value)
     end
 
-    def set_property(property_name, property_value)
+    def set_property(property_name, property_value, property_presentation = property_name)
```

Comment:
Can you set this to `nil` and then use memoization to assign `property_presentation`. I have a gut feeling this might cause a warning in ruby 2.2, but I'm not sure. WDYT?